### ISSUE TITLE
ML스터디 8주차 과제

### ISSUE THREAD

**Issue by whitesoil** [AUSG/AUSG_Study#22]
#오차역전파 알고리즘 (Backpropagation Algorithm)

***다음 항목들을 참고하여 조사하시기 바랍니다. ***
1. 순전파 (Forward propagation)
2. 합성함수
3. 연쇄법칙
4. 역전파 알고리즘
5. Sigmoid, ReLU의 역전파
6. Softmax-with-Loss Layer
